### PR TITLE
Clickable genres in details view

### PR DESCRIPTION
**Pull request by Primdahl26**
### Is your feature request related to a problem? Please describe.

The idea is to have the genres clickable, and after that redirect back to index page, and choose the chosen genre for the user.

### Describe the solution you'd like

The idea is to have the genres clickable, and after that redirect back to index page, and choose the chosen genre for the user.

### CODE REVIEW

**Comment by Primdahl26** [comment]
This PR Makes genres look clickable. 

When clicking a genre you are redirected to index page, with the clicked genre chosen. This also removed any current selected genre, and removed your saved input query.

Also added a clickable looking cursor on hover.

File: frontend/src/components/details/top_card.svelte [hunk 1]
```diff
@@ -37,7 +45,11 @@
             {#if genres}
                 <div class="flex-wrap mt-2">
                     {#each genres as genre}
-                        <div class="badge mx-2">{genre}</div>
+                        <div
+                            on:click={() => changeGenreAndRedirectHome(genre)}
+                            class="badge mx-2 transform hover:scale-110 cursor-pointer">
```
Comment (AndreasPB) [inline frontend/src/components/details/top_card.svelte:50]: I think we should make the badges use `primary`-color to indicate that they are clickable - I think that's how the pro frontenders do it 🙈

**Review by AndreasPB — APPROVED** [review]
Feels really nice, but I think it needs to be primary for people to realize it's clickable without randomly hovering over it

File: frontend/src/components/details/top_card.svelte [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <script lang="ts">
     import ReadMore from "./read_more.svelte";
     import { currentCountry } from '../../stores/country.js';
+    import { currentGenres } from '../../stores/genres.js';
```
[the remark this answers is not in the document]
Comment (Primdahl26) [reply frontend/src/components/details/top_card.svelte:4]: I agree, but not in this PR

**Pull request merged**